### PR TITLE
Add new classroom

### PR DESCRIPTION
**Pull request by simoneduca**
Update edu-api root.
Add createClassroom button dumb component.

### CODE REVIEW

**Comment by simoneduca** [comment]
@rogerhutchings sorry I attempted to squash the commits into one and botched it. But it's ready to review.

**Comment by rogerhutchings** [comment]
Fixes #42

File: src/containers/HeaderAuth.jsx [hunk 1]
```diff
@@ -24,6 +24,10 @@ export default class HeaderAuth extends Component {
   }
 
   login() {
+    console.log('Logging in');
+    // For deploy-preview, use the following Wildcam on staging details in config.json
+    // const appId = '17bdbeb57f54a3bf6344cf7150047879cfa1c8d5f9fd77d64923e6c81fe6e949';
+    // const redirectUri = 'https://preview.zooniverse.org/wge/';
     return Panoptes.oauth.signIn('http://localhost:3000/?env=staging#/');
```
Comment (shaunanoordin) [inline src/containers/HeaderAuth.jsx:31]: Minor query: if we want to deploy a preview to Master by the end of the day, would this need to be addressed with the 'correct' URLs & IDs for the preview version?

Comment (simoneduca) [inline src/containers/HeaderAuth.jsx:31]: I believe so. And then swapped back.

**Comment by shaunanoordin** [comment]
PR Status: **Validated**
Action: **Merging**

**Comment by shaunanoordin** [comment]
WHOOPS added note in the wrong place. Forget this, this didn't happen.

**Pull request merged**